README.md

```diff
@@ -35,27 +70,51 @@ This guide will help you set up Volto-Hydra (AdminUI) and connect your frontend
     make start
     ```
 
-### Step 2: Setting Up Your Frontend
+### Make your frontend editable
 
-**1. Choose Your Framework**
+- Take the latest [hydra.js](https://github.com/collective/volto-hydra/tree/hydra.js) and include it in your frontend
+- Your frontend will know to initialise the hydra iframe bridge when it is being edited using hydra as it will recieve a ```?hydra_auth=xxxxx```
```

Comment:
we should also mention that `hydra_auth` recieved from the bridge be either ' ' or 'xxxxx', meaning they can add it into the plone/client initialization and it will automatically fetch private content if they are loggedIn otherwise it'll be fetching public content. I will add these on Hydra.js documentation.

Reply:
@MAX-786 this is the documenation for hydra.js. Put the instructions in this readme.